src/components/ProjectHeader.js

```diff
@@ -8,15 +8,12 @@ import {
 } from "@trussworks/react-uswds";
 import { useTranslation } from "react-i18next";
 
-const ProjectHeader = () => {
+const ProjectHeader = ({ contactRoute, teamRoute }) => {
   const { t } = useTranslation();
 
   const [expanded, setExpanded] = useState(false);
   const onClick = () => setExpanded((prvExpanded) => !prvExpanded);
 
-  const contactRoute = t("links.contactRoute");
-  const teamRoute = t("links.teamRoute");
-
   const navItems = [
     <Link to={teamRoute}>{t("links.team")}</Link>,
```

Comment:
Is there a reason you are sending the teamRoute through props if you are still going to be using useTranslation() in this class? Does it make it more efficient?

Reply:
Good point I guess the links.teamRoute is basically serving as a variable here. I can just not use a teamRoute variable at all, and directly call t("links.teamRoute") when I need it